app/src/main/java/edu/uc/kovaciad/slicetracker/dto/Model.kt

```diff
@@ -4,12 +4,12 @@ package edu.uc.kovaciad.slicetracker.dto
  * @param id: DB ID for dev use
  * @param modelName: User Inputted Name
  * @param artistId: Artist ID from Artist table
- * @param url: model URL
+ * @param modelURL: model URL
  */
 data class Model (
             var modelName: String = "",
             var artistId: String = "",
-            var url: String? = null,
+            var modelURL: String? = null,
```

Comment:
In the other DTOs the url param is named after the data class (authorURL, brandURL). I suggest changing this param from url to modelURL, to stick with the established naming convention

Reply:
It's OK to keep this as it were, as the context of Model is given by the class name.